src/main/java/org/flockofseagles/Question.java

```diff
@@ -0,0 +1,19 @@
+package org.flockofseagles;
+
+public class Question {
+    private String[] possibleAnswers;
+    private String question;
+
+    public Question(final String[] answers, final int correctAnswerIndex, final String question) {
```

Comment:
Should set `correctAnswerIndex` to a class variable, right?

Reply:
Originally I was going to set it to a class variable, but I thought that it would be simpler to either have the correct answer set as a string, or to just keep the correct answer as the first entry in the String array.

I will add a commit that fixes that.